### DISCUSSION TITLE
posts/2022/280b9d1/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#4] [Announcements]
# posts/2022/280b9d1/

几件今天在想的事情：大类培养、能力断代、社团发展、考研压力、疫情防控、手机游戏。

https://blog.gztime.cc/posts/2022/280b9d1/

**Comment by widcardw** [comment]
太真实了，一直想写一篇关于教育的文章，但都咕了。对于室友起床后lol和王者能坐那玩一整天甚至连饭都不吃的那种情况，我是只能😅了。

**Reply by widcardw** [reply]
从笔者几分钟前的回复来看，感觉可以分析一下从二战结束，也几乎是计算机诞生开始，人们（包括国内外）对于计算机的一些看法了吧，或许能从中得到一些结论吧

**Comment by flwfdd** [comment]
是啊....他们去哪里了啊？大隐隐于市抑或压根不存在了呢....

**Comment by xuantengh** [comment]
有同样的感触。但我觉得跟高中有很大关系：其一是作者提到的由于高考加分政策的变化导致的高中竞赛 (不仅是信息学) 的日薄西山，还有就是并不是所有的高中都能让学生自由地探索，像我一样来自十八线小县城高中的学生在上大学前都没有机会真正系统地理解计算机。

**Reply by xuantengh** [reply]
对于大多数人，要求他们吃力地、一知半解地掌握那些其他有条件的同学已经在高中就轻车熟路的技能后，还要扩展在课外的见闻和兴趣不是那么容易的。

**Reply by xuantengh** [reply]
找到自己热爱的东西是很幸福，但同时也是很难的事情，中大 CS 在这方面确实做得很不好......

**Reply by GZTimeWalker** [reply]
笔者有个困惑是：为什么当年明明没有现在这么好的条件，反而能孕育出很多所谓技术力很强、或是热爱CS的人？

会不会是因为当今选择CS专业的原因更多是能更快赚到更多的钱，从而有兴趣的那些人也不一定能敌过成绩好的人，因此分散在其他的学院中呢？

**Reply by xuantengh** [reply]
一个原因可能是大家对 GPA 的态度变了：一个 14 级的师兄跟我聊过，他们本科的时候还有不少同学以给开源社区做贡献为荣，有不少同学给 `numpy` 和 `scipy` 这类开源库提 commit，对 GPA 和学院课程没有那么重视；现在大家的能力可以说不比当年弱，但学院从大一就开始灌输万般皆下品，唯有“升学”高的理念，而 GPA 无论是在保研还是境外升学过程中都扮演举足轻重的地位，自然而然大家的重心就放在了课内的课程，而学院本科课程的质量也就那样，大家能从中学到的对 CS 的品味和能力的提升可见一斑。

**Comment by Walikrence** [comment]
考虑谈个朋友在校外合租吧，或者申请转宿舍，室友影响真的大

**Reply by GZTimeWalker** [reply]
还好啦，而且不久就会六改四了，会换的w

**Comment by naifenmizuha** [comment]
现在就业形势越来越不乐观，高考筛选的人都可以算太多，大学校园必然成为竞争的延续，棒球队在南校大一也很难招到人......我是觉得硬卷带来的边际效益实在是太少了，卷和不卷遇到的就业形势都差不多，不如学点自己喜欢的东西。

**Comment by xmmmmmovo** [comment]
全世界都卷 但是可润

**Comment by rabbit-dayi** [comment]
真的是这样..

**Comment by Skywt2003** [comment]
感同身受！！作为曾经的 OIer（来自浙江），强基计划的出台（自主招生的取消）让我们搞的竞赛成果付之一炬；咱们大学的 MSC 甚至今年刚被取缔了（因为是「境外组织资助」，就离谱），CTF 也因为保研政策的调整（竞赛加分权重大大降低）而越来越冷门，感觉尽管是信息学院，身边的人大多都是啥都不会，找不到几个大佬。😮‍💨

**Comment by syuy** [comment]
作为第一届参加CSP的OIer目前正在CTF的我深有同感！可能是就业压力大，学历内卷严重使人们不能心安理得地追求个人爱好，去认真钻研某一件事物。回首向来萧瑟处，归去，也无风雨也无晴。向前辈致敬。
